manim/utils/space_ops.py

```diff
@@ -92,7 +106,25 @@ def quaternion_mult(*quats):
         )
 
 
-def quaternion_from_angle_axis(angle, axis, axis_normalized=False):
+def quaternion_from_angle_axis(
+    angle: float, axis: np.ndarray, axis_normalized=False
+) -> np.ndarray:
+    """Gets a quaternion from an angle and an axis.
+
+    Parameters
+    ----------
+    angle : float
+        The angle for the quaternion.
+    axis : np.ndarray
+        The axis for the quaternion
+    axis_normalized : bool, optional
+        Checks whether the axis is normalized, by default False
```

Comment:
you don't need to define types in docstrings if you defined it already as typehints. You have done this everywhere please remove.
```suggestion
    angle
        The angle for the quaternion.
    axis
        The axis for the quaternion
    axis_normalized
        Checks whether the axis is normalized, by default False
```